### ISSUE TITLE
Frequent EBUSY error on changed file

### ISSUE THREAD

**Issue by madbarron** [karma-runner/karma#1213]
When I have the karma server running and I save a tracked file, it often crashes karma with:

```
ERROR [karma]: { [Error: EBUSY, open 'c:\myfile.js']
  errno: 10,
  code: 'EBUSY',
  path: 'c:\\myfile.js' }
Error: EBUSY, open 'c:\myfile.js'
```

I have set `autoWatchBatchDelay: 1000` in my karma.conf.js, but this error occurs before the delay takes place.

I am in Windows 7, using Sublime Text 3, karma 0.12.24, node 0.10.32

**Comment by AbraaoAlves** [comment]
+1
I am in Windows 8.1, using Sublime Text 3, karma 0.12.31, node 0.10.33

**Comment by Narretz** [comment]
I have the same issue. Maybe it was the last Sublime update?

**Comment by backmask** [comment]
Currently, files are loaded as soon as a modification is detected. If a file handle is not released yet, karma crashes (or tests are aborted with 095cfb4dcf6b7520ecbc6e9bcd1d90b7f0fe79f9).

I believe the best fix would be to load modified files after the `autoWatchBatchDelay` It would also have the nice benefit to reload a file once if it is saved multiple times quickly.

**Comment by madbarron** [comment]
Yeah, I expected that karma wouldn't attempt to touch the files until `autoWatchBatchDelay`. Your PR looks like a reasonable workaround.
